src/trans-word-level/sequence.cpp

```diff
@@ -364,42 +364,52 @@ sequence_matchest instantiate_sequence(
 
     return result;
   }
-  else if(expr.id() == ID_sva_sequence_consecutive_repetition) // [*...]
+  else if(expr.id() == ID_sva_sequence_repetition_star) // [*...]
   {
-    // x[*n] is syntactic sugar for x ##1 ... ##1 x, with n repetitions
-    auto &repetition = to_sva_sequence_consecutive_repetition_expr(expr);
-    return instantiate_sequence(
-      repetition.lower(), semantics, t, no_timeframes);
-  }
-  else if(
-    expr.id() == ID_sva_sequence_repetition_plus ||
-    expr.id() == ID_sva_sequence_repetition_star)
-  {
-    // x[+] and x[*]
-    auto &op = to_unary_expr(expr).op();
+    auto &repetition = to_sva_sequence_repetition_star_expr(expr);
+    if(repetition.is_unbounded() && repetition.repetitions_given())
+    {
+      // [*x:$]
+      auto from = numeric_cast_v<mp_integer>(repetition.from());
+      auto &op = repetition.op();
 
-    // Is x a sequence or a state predicate?
-    if(is_SVA_sequence_operator(op))
-      PRECONDITION(false); // no support
+      // Is op a sequence or a state predicate?
+      if(is_SVA_sequence_operator(op))
+        PRECONDITION(false); // no support
```

Comment:
Is this the right way to report this problem, or should this be an exception (with a human-readable error message) instead? (I acknowledge that this is the same behaviour as before.)